tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "./.svelte-kit/tsconfig.json",
   "compilerOptions": {
-    // @rollup/plugin-typescript TS1259: Module '"/Users/justin.lowery/Github/forgerock-web-login-framework/src/locales/ca/en/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag
```

Comment:
This comment was not perfectly accurate, but I made a second attempt at importing JSON into the project without the synthetic import property. Unfortunately, there were too many different tools that I could not get to align when this was removed.